app/styles/components/extras.scss

```diff
@@ -23,6 +23,10 @@ a.link-item {
   color: #555 !important;
 }
 
+.weight-400 {
+  color: #555 !important;
+}
```

Comment:
in form.scss, change .ui.secondary segment color, instead of adding new in weight-400
 
```

.ui.stackable .ui.segments .ui.secondary.segment {
  color: #555 !important;

  &:hover {
    color: #737373 !important;
  }
}
```